### PR TITLE
PyVenv - provide Makefile building a virtual environment

### PR DESCRIPTION
**Pull request by smangels**
* provide a simple Makefile that generates a virtual environment for Python development

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+		python3 -m venv venv ;\
+		source venv/bin/activate ;\
+		pip install --upgrade pip ;\
+		pip install --upgrade -r requirements.txt ;\
+		touch $@ ;\
+	)
+
+test: venv
+	( \
+		source venv/bin/activate ;\
```
Comment (fmartingr) [inline Makefile:16]: Shouldn't this be `.` so it is POSIX compatible? I'm unsure at the moment but commands usually run with `/bin/sh` and it doesn't have the `source` alias.

```suggestion
		. venv/bin/activate ;\
```

**Review by fmartingr — APPROVED** [review]
Hey @smangels, thanks a lot for the contribution! Seems pretty handy to kickstart a development environment and test that everything works correctly. It seems good to go only for one small thing that I have commented out regarding the `source` alias. If you could check that out and confirm that it works we can merge this directly afterwards.

Again, thanks a lot for the contribution, really appreciated!

**Comment by smangels** [comment]
@fmartingr thanks for your contribution, I'm located in homeoffice since 2 years ago and trying to develop a PI based solution where it fetches my busy state from O365 and indicates the state using a Luxafor LED at my office's door. :-) So a PI Zero2 should be more than enough I guess. Maybe it could expose an API using FLASK so that I could hit the Do-Not-Disturb button sending our a POST request :-)

**Comment by fmartingr** [comment]
> @fmartingr thanks for your contribution, I'm located in homeoffice since 2 years ago and trying to develop a PI based solution where it fetches my busy state from O365 and indicates the state using a Luxafor LED at my office's door. :-) So a PI Zero2 should be more than enough I guess. Maybe it could expose an API using FLASK so that I could hit the Do-Not-Disturb button sending our a POST request :-)

That sounds like an awesome idea to avoid getting interrupted in meetings and so on. A Pi Zero should be more than enough I think, you only need internet access, a running service and the Luxa connected via USB, so it will have resources to spare anyway :)

**Pull request merged**